IFC4x3/Templates/Product Shape/Product Geometric Representation/Alignment Geometry/Documentation.md

```diff
@@ -0,0 +1,4 @@
+Base for a standard two (Alignment Geometry Gradient) and three (Alignment Geometry Gradient) layout alignment geometric representation.
```

Comment:
It states `Alignment Geometry Gradient` twice - intentional?

Reply:
I think it should be:
```suggestion
Base for a standard two (Alignment Geometry Gradient) and three (Alignment Geometry Cant) layout alignment geometric representation.
```

maybe @SergejMuhic

Reply:
I will look into this.

Reply:
Addressed.